### ISSUE TITLE
$(CurrentFileFullPath) variable points to wrong file

### ISSUE THREAD

**Issue by bigangrydog** [eranif/codelite#1149]
When right-clicking on a file in the workspace/project list, the $(CurrentFileFullPath) is the path of the file open in the editor, and the not file selected with the right-click.

Codelight version 9.1.1

**Comment by eranif** [comment]
Its actually pointing the correct file - this variable expands to the current file opened in the editor and not the one selected by the right click menu. If you require different macro, please open a feature request

**Comment by bigangrydog** [comment]
I'm suggesting that when the user deliberately right-clicks and performs an operation on a file, then $(CurrentFileFullPath) should point to that file for that specific operation.

I've attached a screenshot to show what I mean. In this, I've selected the util.cpp file and right-clicked complile (I'm passing CurrentFileFullPath into a makefile here). Instead of compiling util.cpp, the makefile has been supplied with the filename in editor, when in fact, what is open in the editor is irrelevant in this case.

![right click](https://cloud.githubusercontent.com/assets/17156238/12947776/c9d24bfc-cff5-11e5-80db-d19b18c16e4c.jpg)

Adding a new variable, say $(ContextFileFullPath), would only introduce ambiguity with calls driven from the main menu. If you decide not to implement the change, then an alternative would be to simply remove the right-click context menus for the files in the project tree, as it is currently not functioning as one would expect.
